build.gradle

```diff
@@ -74,34 +89,46 @@ dependencies {
     compile "org.grails.plugins:scaffolding"
     compile "org.grails.plugins:events"
     compile "org.grails.plugins:gsp"
+    compileOnly "io.micronaut:micronaut-inject-groovy"
     console "org.grails:grails-console"
-    compile "org.codehaus.groovy:groovy:2.5.6"
-    compile "org.codehaus.groovy:groovy-templates:2.5.6"
+//    compile "org.codehaus.groovy:groovy:2.5.6"
+//    compile "org.codehaus.groovy:groovy-templates:2.5.6"
     profile "org.grails.profiles:web"
 
     compile "commons-fileupload:commons-fileupload:1.3.1"
 
     runtime "org.glassfish.web:el-impl:2.1.2-b03"
     runtime "org.apache.tomcat:tomcat-jdbc"
-    runtime "com.bertramlabs.plugins:asset-pipeline-grails:2.15.1"
+    runtime "javax.xml.bind:jaxb-api:2.3.1"
+    runtime "com.bertramlabs.plugins:asset-pipeline-grails:3.2.4"
+    testCompile "io.micronaut:micronaut-inject-groovy"
     testCompile "org.grails:grails-gorm-testing-support"
-    testCompile "org.grails.plugins:geb"
+    testCompile "org.mockito:mockito-core"
     testCompile "org.grails:grails-web-testing-support"
-    testCompile "org.grails.plugins:geb:1.1.2"
-    testRuntime "org.seleniumhq.selenium:selenium-htmlunit-driver:2.47.1", withoutXerces
-    testRuntime "net.sourceforge.htmlunit:htmlunit:2.18"
-    testRuntime "org.seleniumhq.selenium:selenium-chrome-driver:2.47.1"
+    testCompile "org.grails.plugins:geb"
+    testCompile "org.seleniumhq.selenium:selenium-remote-driver:3.14.0"
+    testCompile "org.seleniumhq.selenium:selenium-api:3.14.0"
+    testCompile "org.seleniumhq.selenium:selenium-support:3.14.0"
+    testRuntime "org.seleniumhq.selenium:selenium-chrome-driver:3.14.0"
+    testRuntime "org.seleniumhq.selenium:selenium-firefox-driver:3.14.0"
+
+//    testRuntime "org.seleniumhq.selenium:selenium-htmlunit-driver:2.47.1", withoutXerces
+//    testRuntime "net.sourceforge.htmlunit:htmlunit:2.18"
+//    testRuntime "org.seleniumhq.selenium:selenium-chrome-driver:2.47.1"
 
 
     // app specific dependencies
-    compile "org.grails.plugins:ala-auth:3.1.2"
-    compile "org.grails.plugins:ala-ws-security-plugin:2.0"
-    compile ("org.grails.plugins:ala-ws-plugin:2.0") {
+    compile "org.grails.plugins:ala-auth:3.2.3"
+    compile "au.org.ala:userdetails-service-client:1.5.0"
+    compile "org.codehaus.groovy.modules.http-builder:http-builder:0.7.1"
+    compile "org.grails.plugins:ala-ws-security-plugin:2.0.1"
+    compile ("org.grails.plugins:ala-ws-plugin:2.1") {
         exclude module: 'asset-pipeline-grails'
         exclude module: 'xerces'
     }
 
-    compile ('au.org.ala:ala-name-matching:3.4') {
+    compile ('au.org.ala:ala-name-matching-search:4.2')
+    compile ('au.org.ala:ala-name-matching:4.2') {
```

Comment:
You should be able to remove this dependency completely, since ala-name-matching-search (without lucene) should be all you need (?)

Reply:
both are needed.